sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/ComponentsTests.java

```diff
@@ -48,7 +49,7 @@ public void componentLifecycleTest(HttpClient httpClient, DigitalTwinsServiceVer
             List<ModelData> createdList = client.createModels(modelsList);
             logger.info("Created {} models successfully", createdList.size());
 
-            BasicDigitalTwin createdTwin = client.createDigitalTwin(roomWithWifiTwinId, roomWithWifiTwin,BasicDigitalTwin.class);
```

Comment:
Note that the return type is different than the passed in type (BasicDigitalTwin vs string)